sympy/physics/mechanics/pathway.py

```diff
@@ -41,6 +32,24 @@ class PathwayBase(ABC):
     Instances of this class cannot be directly instantiated by users. However,
     it can be used to created custom pathway types through subclassing.
 
+    All pathways use the same sign conventions for the relative speed of and
+    forces acting on the attachment points. Take attachment points ``a1`` and
+    ``a2`` that move along the same line and have an equal and opposite force
+    of magnitude ``F``. If we define the positive sense of the velocity of each
+    point to the right (positive ``v1`` or ``v2`` describe rightward motion)
+    and the equal and opposite force ``F`` has a sign convention such that
+    positive force is in the same direction for ``a2`` and opposite direction
+    as ``a1``, respectively, then we have the following sign convention::
+
+       a1          a2
+       o<--- F --->o
+       |           |
+       --> v1      --> v2
+
+    We define the relative speed, or extension velocity,  as ``v2 - v1`` with a
+    positive ``F`` tending to move ``a2`` the right and ``a1`` to the left,
+    i.e. expanding if ``a2``'s position is to the right of ``a1``'s.
```

Comment:
Here is a first attempt at trying to say something about the sign convention.

As you can probably see, "extension" and "expansion" don't necessarily have any meaning unless you make the assumption that a2 is always positioned to the right of a1. But the equations present in these classes are valid even if a1 is the right of a2.

Reply:
I'm seeing now that your "extension_vel" means "when the two points are moving away from each other". Here's a little sign table:

```python
    def test_2D_pathway_extension_velocity(self) -> None:
        """

        q1  q1d  extension_vel
        +   +    +
        +   -    -
        -   +    -
        -   -    +

        Distance between points is growing: + extension_vel
        Distance between points is shrinking: - extension_vel

        """
        self.pB.set_pos(self.pA, 2*self.q1*self.N.x)
        expected = 2*self.q1*self.q1d/sqrt(self.q1**2)
        assert self.pathway.extension_velocity == expected
```

Your sign convention is a function of position, which strikes me as odd at this moment. I'll think about it more in the morning.

Reply:
I think we've got this explained clearly now.